Gems/AWSMetrics/cdk/api_spec.json

```diff
@@ -68,7 +68,7 @@
                         "content": {
                             "application/json": {
                                 "schema": {
-                                    "$ref": "#/components/schemas/MetricsEventSuccessResponse"
+                                    "$ref": "#/components/schemas/PostMetricsEventsResponse"
```

Comment:
As you changed api json, what will happen if customer has already deployed resources based on your old api json. I know it is less likely we have customer use it, but this seems like a backward incompatible change?

Any process we should take here to guide customer?

Reply:
Good point but I don't think there's backward incompatible issue here. This PR only renames the models used by the API but didn't change the API call itself, i.e. you can still call the endpoint deployed by your previous CDK application deployment without any modification. Updating your existing CDK application with CDK CLI shouldn't cause any error as well.

Reply:
If customer deployed resources with old api json, which is using the older reference name. And once they checkout your cleanup code, it can work without modifying existing resource? Can you test it?

For example. you change this reference from MetricsEventSuccessResponse to PostMetricsEventsResponse. And in c++ code you also changed the class to PostMetricsEventsResponse, then how old resource model won't have PostMetricsEventsResponse?

Reply:
Or reference name just works like a helper function to hold model details within server api gateway scope only, it doesn't get used to refer across client and server side?

Reply:
The reference name just works like a helper function to hold model details within server api gateway scope only. You can use different names on the client and server side and just format the request body following the schema. I can actually keep  api json as it was without any modification and only change the client side class names but I just update it to keep the consistency and readability.

Sure. I can test the case you talked about and comment on this PR later.